### DISCUSSION TITLE
Approved but no benefits since 14th May

### DISCUSSION THREAD

**Discussion by DanVasii** [community/community#55763] [GitHub Education]
### Hi everyone,

I have been waiting since 14th May for my benefits but they didn' arrive yet. From what i read, it should take about 72 hours after the verification to ge the email with benefits. I have opened a ticket but no replies yet from github. What should I do next ?

**Comment by Gianiboy2265** [comment]
I have the exact same issue after 72 hours no benefits

**Comment by Ntshangase** [comment]
Hey @DanVasii 

Your student benefits have been awarded🎉. You can login to [Global Campius](https://education.github.com/globalcampus/student) and start using your student benefits.

You can verify your application status here: [Verify Application Status](https://education.github.com/discount_requests/application)

**Reply by Angelakeke** [reply]
I have the same issue. Should I check something? Thanks.

**Reply by Ntshangase** [reply]
You can check your application status here [Click Here](https://education.github.com/discount_requests/application)

**Reply by Gianiboy2265** [reply]
For me it is still approved and no benefits

**Reply by Angelakeke** [reply]
It is still approved and no benefits + 1

**Reply by Ntshangase** [reply]
You should be getting your benefits soon, If you haven't gotten them after 72hours then I would suggest you open a separate discussion.

**Reply by MiskaWasTaken** [reply]
same issue here, continuing to wait

**Reply by Alohaq** [reply]
It is still approved and no benefits + 1

**Comment by samus-aran** [comment]
Hey @DanVasii  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. **However, during busy periods like the start of a new semester, it may take a bit longer.**

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.

**Comment by WBilly-dev** [comment]
Hi everyone,
I have been waiting since 30th September for my benefits but they didn' arrive yet. From what i read, it should take about 72 hours after the verification to ge the email with benefits.  What should I do next ?

**Reply by M1hono** [reply]
Have you get benifits now?

**Reply by samus-aran** [reply]
@M1hono 

When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

We would recommend to create a new discussion if you haven't received your academic benefits within 5 days since your approved status.

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
